resources/qml/TimelineView.qml

```diff
@@ -217,9 +208,12 @@ Page {
 						text: chat.model.formatDateSeparator(modelData.timestamp)
 						color: colors.brightText
 
-						height: contentHeight * 1.2
-						width: contentWidth * 1.2
```

Comment:
This shouldn't be producing a binding loop. Does it actually do that on your system? If it does, it probably would still make more sense to define the padding by the font size instead of static sizes. (This makes the pills giant for me!).

Reply:
![image](https://user-images.githubusercontent.com/856493/80565091-7f345780-89a4-11ea-8aac-80b5fc0eb2c2.png)

I was hoping Qt scales its "pixels" automagically for screen densities, but maybe I was wrong. I'll see if I can figure out a way of pre-determining font relative heights